qt5/meta.yaml

```diff
@@ -18,9 +18,11 @@ requirements:
     - freetype # [unix]
     - jom # [win]
     - python # [win]
+    - libxcb # [linux]
     - icu
```

Comment:
I think you should add here `gcc` as a build dependency. `gcc` is now on the Continuum repos and it's at version 4.8.2.

fontconfig seems to be missing as a build dependency too.